Prep/Source/Alignment/ICPRigid3DImageRegistration/source/ICPRigid3DImageRegistration.cxx

```diff
@@ -314,10 +332,10 @@ int main(int argc, char * argv [] )
         InterpolatorType::Pointer interpolator = InterpolatorType::New();
 
         resampler->SetInterpolator( interpolator );
```

Comment:
Rather than using nearest neighbor and directly resample the segmention, the resampler of the segmentation should resample the distance transform using linear interpolation and then threshold to resampled distance transform to produce the transformed segmentation. For now, please submit an issue and we could work this out later.